FWCore/Framework/test/stubs/TestStreamProducers.cc

```diff
@@ -500,7 +500,7 @@ namespace edmtest {
         }
       }
 
-      static std::shared_ptr<UnsafeCache> accessInputProcessBlock(edm::ProcessBlock const&, TestGlobalCache const*) {
+      static std::shared_ptr<UnsafeCache> accessInputProcessBlock(edm::ProcessBlock const&, TestGlobalCache*) {
```

Comment:
Does this get called? I thought it required `edm::InputProcessBlockCache<>` to be used?

Reply:
In this PR, it does not get called. When the next PR which implements persistence is submitted, it will get called and there will be meaningful tests.